### PR TITLE
feat: add header menu 

### PR DESCRIPTION
**Pull request by amt-augustineopokukesse**
Created the menu section of the header

### CODE REVIEW

File: apps/iglidur-designer/src/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
       <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" />
```
Comment (Bilal-Abubakari) [inline apps/iglidur-designer/src/index.html:11]: Please what is this for?

File: apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.html [hunk 2]
```diff
@@ -1,6 +1,35 @@
 <div class="header">
-    <div class="logo">
-        
+  <div class="logo">
+    <img src="assets/igus-logo.svg" alt="" />
+  </div>
+  <div class="menu-section">
+    <div class="menu-options">
+        <div class="menu-option"><p><i class="fa fa-envelope"></i></p><p>Contact</p></div>
```
Comment (Bilal-Abubakari) [inline apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.html:7]: Use angular material and not fontawesome

**Review by Bilal-Abubakari — COMMENTED** [review]
It will also be nice to provide a visual image in your PR description

**Review by Bilal-Abubakari — COMMENTED** [review]
You should be using scss and not css

File: apps/iglidur-designer/src/index.html (hunk 1 above)
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/index.html:11]: For adding icons

File: apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.html (hunk 2 above)
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.html:7]: Okay

**Comment by amt-augustineopokukesse** [comment]
Okay.. will switch from css to scss

**Pull request merged**